### PR TITLE
DevTools 107 > Settings > Shortcuts

### PR DESCRIPTION
**Pull request by sofiayem**
Documented:
- https://developer.chrome.com/blog/new-in-devtools-107/#shortcuts
- https://developer.chrome.com/blog/new-in-devtools-107/#toggle-themes

Publish on Oct 25.

### CODE REVIEW

**Review by jecfish — APPROVED** [review]

File: site/en/docs/devtools/customize/index.md [hunk 1]
```diff
@@ -93,20 +93,43 @@ You can sync your DevTools settings across devices. To enable that, you need to:
 2. The DevTools sync settings can be updated via **Settings** > **Sync** > **Enable settings sync** checkbox.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LUwFNTDyP22L1euSGg73.png", alt="DevTools sync settings", width="800", height="654" %}
 3. Most of the settings are synced, except for the **Workspace**, **Experiments**, and **Devices** tabs and a few other general settings. The state of the **Enable settings sync** checkbox is synced across devices as well.
-  For example, the following **appearance** settings are synced so you have a consistent experience across devices and don’t need to re-define the same settings again.
+  For example, the following **appearance** settings are synced so you have a consistent experience across devices and don't need to re-define the same settings again.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/t8SQuZ4mE2xiLVxaZz11.png", alt="appearance settings", width="800", height="584" %}
```
Comment (rachelandrew) [inline site/en/docs/devtools/customize/index.md:97]:
```suggestion
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/t8SQuZ4mE2xiLVxaZz11.png", alt="The appearance settings.", width="800", height="584" %}
```

File: site/en/docs/devtools/customize/index.md [hunk 2]
```diff
@@ -93,20 +93,43 @@ You can sync your DevTools settings across devices. To enable that, you need to:
...
-  However, the **dock** settings isn’t sync because developers have different dock preferences when debugging on different sites.
+  However, the **dock** settings isn't sync because developers have different dock preferences when debugging on different sites.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/jWv8rwkF4q6SwTQbSNpp.png", alt="dock", width="426", height="134" %}
```
Comment (rachelandrew) [inline site/en/docs/devtools/customize/index.md:99]:
```suggestion
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/jWv8rwkF4q6SwTQbSNpp.png", alt="The dock.", width="426", height="134" %}
```

File: site/en/docs/devtools/customize/index.md [hunk 3]
```diff
@@ -93,20 +93,43 @@ You can sync your DevTools settings across devices. To enable that, you need to:
...
 
 
 ## Customize keyboard shortcuts {: #keyboard-shortcuts }
 
-See [Customize chords keyboard shortcuts](/blog/new-in-devtools-88/#keyboard-shortcuts).
+To edit keyboard shortcuts:
 
-{% Img src="image/admin/tF18281gSqF69MENLGuk.png", alt="Chords keyboard shortcuts", width="800", height="508" %}
+1. [In DevTools](/docs/devtools/open/), click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings", width="24", height="24" %} **Settings**.
```
Comment (rachelandrew) [inline site/en/docs/devtools/customize/index.md:106]:
```suggestion
1. [In DevTools](/docs/devtools/open/), click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} **Settings**.
```

**Review by rachelandrew — APPROVED** [review]
Just a couple of alt text nits.

**Pull request merged**